src/pymedphys/docker/orthanc/scripts/morph_header_and_forward.lua

```diff
@@ -0,0 +1,50 @@
+-- Based on https://bitbucket.org/osimis/orthanc-setup-samples/src/c05537f4c4b66dc622356b15c67d40a78fcb476c/docker/transcode-middleman/orthanc-middleman/transcodeAndForward.lua?at=master&fileviewer=file-view-default
+
+function OnStoredInstance(instanceId, tags, metadata, origin)
+  -- Do not compress twice the same file
+  if origin['RequestOrigin'] ~= 'Lua' then
+    -- Retrieve the incoming DICOM instance from Orthanc
+    local dicom = RestApiGet('/instances/' .. instanceId .. '/file')
+
+    -- Write the DICOM content to some temporary file
+    local received_filepath = instanceId .. '-received.dcm'
+    local converted_filepath = instanceId .. '-converted.dcm'
+
+    local plan_UID = '1.2.840.10008.5.1.4.1.1.481.5'
+    local structure_UID = '1.2.840.10008.5.1.4.1.1.481.3'
+
+
+    print(tags['SOPClassUID'])
+
+    if tags['SOPClassUID'] == structure_UID or tags['SOPClassUID'] == plan_UID then
+      local target = assert(io.open(received_filepath, 'wb'))
+      target:write(dicom)
+      target:close()
+
+      RestApiDelete('/instances/' .. instanceId)
+    end
+
+    if tags['SOPClassUID'] == structure_UID then
+      print('Converting a structure set')
+      os.execute('pymedphys dicom adjust-rel-elec-density ' .. received_filepath .. ' ' .. converted_filepath .. ' "Couch Edge" 1.1 "Couch Foam Half Couch" 0.06 "Couch Outer Half Couch" 0.5')
```

Comment:
Here are some hard coded couch values. I'm still finding my feet with the Lua language which is what Orthanc uses for scripting. In the future these hardcoded values need to be pulled out into a configuration file.